Assets/MixedRealityToolkit/InputSystem/Cursors/BaseCursor.cs

```diff
@@ -133,11 +134,9 @@ public bool IsVisible
         /// <param name="eventData"></param>
         public virtual void OnSourceDetected(SourceStateEventData eventData)
         {
+            var controller = eventData.Controller as WindowsMixedRealityController;
 #if UNITY_WSA
```

Comment:
Should only be using the input system (I know this is on the TODO)

Reply:
We can get the data directly VIA the interactionMapping!!!!

Reply:
Oh, fun fact - Source State changes do not have actions.